README.md

```diff
@@ -18,9 +18,9 @@ If you are new to splot and PySAL you will best get started with our [documentat
 
 ## Installing splot
 
-### splot requires:
+### Installing dependencies:
 
-splot is compatible with `Python` 3.5 or later and depends on `geopandas` 0.4.0 or later and `matplotlib` 2.2.2 or later.
+splot is compatible with `Python` 3.6 or later and depends on `geopandas` 0.4.0 or later and `matplotlib` 2.2.2 or later.
```

Comment:
Maybe want to say just 3.6 and 3.7 in the off-chance an early 3.8 adopter tries `splot` and something breaks?